libkbfs/mdserver_memory.go

```diff
@@ -408,9 +431,24 @@ func (md *MDServerMemory) GetRange(ctx context.Context, id tlf.ID,
 		return nil, err
 	}
 
-	md.lock.Lock()
-	defer md.lock.Unlock()
-	return md.getRangeLocked(ctx, id, bid, mStatus, start, stop, lockBeforeGet)
+	for {
+		rmds, ch, err := md.doGetRange(
+			ctx, id, bid, mStatus, start, stop, lockBeforeGet)
+		if err != nil {
+			return nil, err
+		}
+		if ch == nil {
+			return rmds, err
+		}
+		select {
```

Comment:
`mdserver_remote` would get a throttled error here and rely on `rpc` to retry. So maybe include a comment here that we are doing the RPC retry logic here within the handler? Same for `Lock` below.